src/window_selectable.cpp

```diff
@@ -133,15 +133,13 @@ void Window_Selectable::Update() {
 	Window_Base::Update();
 	if (active && item_max > 0 && index >= 0) {
 		if (Input::IsRepeated(Input::DOWN) || Input::IsTriggered(Input::SCROLL_DOWN)) {
-			if (index < item_max - column_max || (column_max == 1 &&
-				(Input::IsTriggered(Input::DOWN) || Input::IsTriggered(Input::SCROLL_DOWN)))) {
+			if (index < item_max - column_max || column_max == 1) {
 				Game_System::SePlay(Game_System::GetSystemSE(Game_System::SFX_Cursor));
 				index = (index + column_max) % item_max;
 			}
 		}
 		if (Input::IsRepeated(Input::UP) || Input::IsTriggered(Input::SCROLL_UP)) {
-			if (index >= column_max || (column_max == 1 &&
-				(Input::IsTriggered(Input::UP) || Input::IsTriggered(Input::SCROLL_UP)))) {
+			if (index >= column_max || column_max == 1) {
```

Comment:
For some reason, this feature doesn't work in the actor battle command window in 2k3 only. It works everywhere else I tried it.

That being said, this is an RPG_RT bug I don't care to emulate.